.github/workflows/build-and-deploy-production.yml

```diff
@@ -0,0 +1,146 @@
+name: Deploy Auctions UI production
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
We previously used production release trigger, as it is done for electron app:
https://github.com/sidestream-tech/unified-auctions-ui/blob/3596555b94f73a3aea3d4481e2b82d4606d879d5/.github/workflows/build-electron-app.yaml#L3-L5

Why do you prefer to trigger it on tag creation instead? What's the main difference?

Reply:
No main difference as the outcome is the same for both :) I moved more to the `push on tag` approach as it allowed me to run test deployments with different tags. As i dont think i will do this on this project a lot its totally fine for me to use ur provided approach :D